README.md

```diff
@@ -213,6 +213,8 @@ These are full-blown game engines.
 * [OctaForge][379] - Implemented in C++. Has LuaJIT for scripting. Fork of
   Sauerbraten. [NCSA Open Source License][380].
 * [Ogre3D][41] - Implemented in C++. Has a [range of bindings][42]. [Expat][11].
+* [OHRRPGCE][421] - Implemented in BASIC. Designed for 8/16-bit RPG games like Final Fantasy.
+  Has custom plotscript language for advanced scripts. An oldschool RPG Maker alternative. [GNU GPLv2][422] or later.
```

Comment:
Link to the GPLv2 reference that's already present to avoid duplication:

```suggestion
  Has custom plotscript language for advanced scripts. An oldschool RPG Maker alternative. [GNU GPLv2][14] or later.
```

Reply:
Ok, I understand. I just wanted to point to the OHRRPGCE's wiki page about license if anyone is concerned.